<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceEditor</class>
 <widget class="QWidget" name="ResourceEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1077</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ResourceEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="8,2">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="listWgt"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="btnCreate">
             <property name="text">
              <string>Создать</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOpen">
             <property name="text">
              <string>Открыть</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemove">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QTreeWidget" name="treeWgt">
           <column>
            <property name="text">
             <string notr="true">Префикс</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Превдоним</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Свойства</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Псевдоним:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditAlias"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Префикс:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lnEditPrefix"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btnAddItem">
             <property name="text">
              <string>Добавить узел</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAddImgItem">
             <property name="text">
              <string>Добавить файл</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnRemoveItem">
             <property name="text">
              <string>Удалить</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>206</width>
            <height>352</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lblReview">
             <property name="text">
              <string>Просмотр</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lblRegister">
       <property name="text">
        <string>Не зарегистрирован</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>768</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegister">
       <property name="text">
        <string>Регистровать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
